packages/opencensus-web-instrumentation-zone/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@opencensus/web-instrumentation-zone",
+  "version": "0.0.2",
+  "description": "OpenCensus Web instrumentation zone to generate user interaction spans for interactions after first page loads.",
```

Comment:
Rather than "... instrumentation zone to generate ..." could it be "... instrumentation zone is used to generate ..." (I think it's just missing a phrase there)?